tests/scripts/.definitions.sh

```diff
@@ -17,7 +17,7 @@ TERRAFORM="terraform -chdir=${TERRAFORM_DIR}"
 : ${LOG_DIR:="/tmp/logs"}
 : ${PROJECT:="$(basename "${PROJECT_DIR}")"}
 : ${TEST_NAMESPACE:="test-operator"}
-: ${TARGET_DRIVER_VERSION:="535.183.06"}
+: ${TARGET_DRIVER_VERSION:="560.35.03"}
```

Comment:
Question: Do we want to always test the default driver version in the operator helm chart?

Reply:
This is the target driver version used for the driver upgrade tests in our e2e suite

Reply:
The default driver version for the e2e tests is taken from the values.yaml which is R550 currently